modulefiles/post/v8.0.0-hera

```diff
@@ -36,13 +36,16 @@ module load g2/3.1.1
 module load gfsio/1.1.0
 module load ip/3.0.2
 module load sp/2.0.3
-module load w3emc/2.3.1
+#module load w3emc/2.3.1
 module load w3nco/2.0.7
 module load crtm/2.2.5
 #module load netcdf/3.6.3
-module load netcdf/4.7.0
+#module load netcdf/4.7.0
 module load g2tmpl/1.6.0
 module load wrfio/1.1.1
+module load hdf5_parallel/1.10.6
```

Comment:
add "module use /scratch2/NCEPDEV/nwprod/NCEPLIBS/modulefiles" to use these new modules

Reply:
This module path is already added at the line 24 in v8.0.0-hera.

Reply:
I see. The view window had that cut off.